### PR TITLE
Dump to h2 command

### PR DESCRIPTION
**Pull request by ogeagla**
Dumps app db to H2 file. 

Test this as follows:
```lein run dump-to-h2 "/path/to/h2.db"```
Validate with:
```lein run load-from-h2 "/path/to/h2.db"```

You'll want to use a non-H2 db as the app db so add these to commands:
```MB_DB_TYPE=postgres MB_DB_HOST=localhost MB_DB_PORT=5432 MB_DB_USER=myuser MB_DB_DBNAME=mydbname```

### CODE REVIEW

File: src/metabase/cmd/dump_to_h2.clj [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+            :let [rows (jdbc/query db-conn [(str "SELECT * FROM " (name table-name))])]
+            :when (seq rows)]
+      (insert-entity! target-db-conn e rows))))
+
+(defn- get-target-db-conn [h2-filename]
+  (h2-details h2-filename))
+
+;;; --------------------------------------------------- Public Fns ---------------------------------------------------
+
+(defmacro dowhile->
```
Comment (camsaul) [inline src/metabase/cmd/dump_to_h2.clj]: are you using this macro anywhere?

Comment (ogeagla) [reply src/metabase/cmd/dump_to_h2.clj]: Ah, not anymore

File: src/metabase/cmd.clj [hunk 2]
```diff
@@ -37,6 +37,15 @@
    (binding [mdb/*disable-data-migrations* true]
      ((resolve 'metabase.cmd.load-from-h2/load-from-h2!) h2-connection-string))))
 
+(defn ^:command dump-to-h2
+  "Transfer data from existing database to newly created H2 DB."
+  ([h2-filename]
```
Comment (camsaul) [inline src/metabase/cmd.clj]: Style thing: you shouldn't wrap the bindings+body in a list as if the fn had multiple arities if it only has a single arity

**Review by camsaul — APPROVED** [review]
other than minor style thing I think this is g2g

**Pull request merged**